k8s/migration/internal/controller/migrationplan_controller.go

```diff
@@ -152,9 +157,103 @@ func (r *MigrationPlanReconciler) reconcileDelete(
 	return ctrl.Result{}, nil
 }
 
-// Similar to the Reconcile function above, but specifically for reconciling the Jobs
+func (r *MigrationPlanReconciler) reconcilePostMigration(ctx context.Context, scope *scope.MigrationPlanScope, vm string) error {
+	migrationplan := scope.MigrationPlan
+	log := scope.Logger
```

Comment:
Don't use this. we have this 
```suggestion
	ctxlog := log.FromContext(ctx).WithName(constants.MigrationControllerName)
```